pkg/memory-monitor/src/monitor/cgroups.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef MM_CGROUPS_H
+#define MM_CGROUPS_H
+
+#define CGROUP_PATH_PREFIX "/sys/fs/cgroup/memory"
+#define EVE_CGROUP         "eve"
+#define PILLAR_CGROUP      "eve/services/pillar"
```

Comment:
Why not constants instead of macros? during debugging building, you'll see string, not variable.

Reply:
And why is it better?

Reply:
No magical numbers, constants are handled by compiler so type safety

Reply:
And here are also no magic numbers... We name these constants. And we know their type...